group_vars/dev/all.yaml

```diff
@@ -52,4 +54,4 @@ profiles:
     # any additional scripts - e.g. clone specific repo
     commands:
         - # git clone repo 1
-        - # git clone repo 2
\ No newline at end of file
+        - # git clone repo 2
```

Comment:
could you add ` || true` to the end of the git clone commands?
This means it won't try to re-clone a repository if a user already has the repos from a previous session